src/v2xds.rs

```diff
@@ -88,6 +88,8 @@ pub struct Metadata {
 #[derive(Serialize, Deserialize, Debug)]
 pub struct LbFilterMetadata {
     pub canary: bool,
+    pub revision: String,
+    pub task_arn: String,
```

Comment:
This field name should be named "instance_id" because the registration API receives "instance_id" https://github.com/cookpad/sds#registration (I don't know why "task_arn" name comes here)

Reply:
It is because we actually put `task_arn` to `instance_id` column and I confused when I see first time. I would like to change the `instance_id` column name to `task_arn` in the future. This is the first step to rename. 
But I may not change the column name for our use case because this is OSS.
@eagletmt Should I keep `instance_id` column name?

Reply:
https://github.com/cookpad/sds/pull/16/commits/f0febda6dbc824908c295a70c16f66dbbcd77576 Kept instance_id